app/models/container.rb

```diff
@@ -4,5 +4,6 @@ class Container < ActiveRecord::Base
 
   belongs_to :container_group
   delegate   :ext_management_system, :to => :container_group
+  delegate   :container_project, :to => :container_group
```

Comment:
@abonas this should be `:through` right? (Also above, but that would be another patch)